lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -74,12 +74,12 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
             path: anchor.devtoolsNodePath || '',
             selector: anchor.selector || '',
             nodeLabel: anchor.nodeLabel || '',
-            snippet: anchor.outerHTML || '',
+            snippet: anchor.snippet || '',
```

Comment:
can we update the PR title to reflect these changes too?

if can't fix, can mention in the body.